Feature/bender/psa 26753 finalise terraform provider (#1)
DigitecGalaxus/terraform-provider-dg-nservicebus: docs/resources/dgservicebus_endpoint.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "dgservicebus_endpoint Resource - terraform-provider-dg-servicebus"
subcategory: ""
description: |-
---

# dgservicebus_endpoint (Resource)

> The Endpoint resource allows the creation and management of a NServicebSu Endpoint.

Reviewer: ```suggestion
The Endpoint resource allows consumers to create and manage an NServiceBus Endpoint.
```

> While creating the endpoint and adding subscribers, a deny-all default rule ensures that no invalid messages are received during creation.

Reviewer: ```suggestion
When initially creating the Endpoint, a default deny-all rule ensures that no invalid messages are received, before the configured subscription rules have been applied.
```
